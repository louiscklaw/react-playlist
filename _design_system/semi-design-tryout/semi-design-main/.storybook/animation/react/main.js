const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
