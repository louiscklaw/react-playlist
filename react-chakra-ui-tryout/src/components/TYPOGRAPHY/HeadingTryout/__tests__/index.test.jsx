import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { HeadingTryout } from '../';

describe('test HeadingTryout tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<HeadingTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
