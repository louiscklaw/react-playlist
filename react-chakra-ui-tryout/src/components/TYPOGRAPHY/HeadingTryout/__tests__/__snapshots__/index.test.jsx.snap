// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test HeadingTryout tryout helloworld 1`] = `
<div>
  HeadingTryout
  <div>
    <h2
      className="chakra-heading css-0"
    >
      I'm a Heading
    </h2>
  </div>
  <div>
    <div
      className="chakra-stack css-16voevi"
    >
      <h1
        className="chakra-heading css-8uhtka"
      >
        (4xl) In love with React & Next
      </h1>
      <h2
        className="chakra-heading css-8uhtka"
      >
        (3xl) In love with React & Next
      </h2>
      <h2
        className="chakra-heading css-0"
      >
        (2xl) In love with React & Next
      </h2>
      <h2
        className="chakra-heading css-0"
      >
        (xl) In love with React & Next
      </h2>
      <h3
        className="chakra-heading css-0"
      >
        (lg) In love with React & Next
      </h3>
      <h4
        className="chakra-heading css-0"
      >
        (md) In love with React & Next
      </h4>
      <h5
        className="chakra-heading css-0"
      >
        (sm) In love with React & Next
      </h5>
      <h6
        className="chakra-heading css-0"
      >
        (xs) In love with React & Next
      </h6>
    </div>
  </div>
  <div>
    <h2
      className="chakra-heading css-8uhtka"
    >
      Basic text writing, including headings, body text, lists, and more.
    </h2>
  </div>
  <div>
    <h2
      className="chakra-heading css-1ud2ify"
    >
      I'm overriding this heading
    </h2>
  </div>
  <div>
    <div
      className="css-19t51sz"
    >
      <h2
        className="chakra-heading css-1itv5e3"
      >
        Modern online and offline payments for Africa
      </h2>
      <p
        className="chakra-text css-6dvxm2"
      >
        Paystack helps businesses in Africa get paid by anyone, anywhere in the world
      </p>
      <button
        className="chakra-button css-qh0qbs"
        type="button"
      >
        Create a free account
      </button>
    </div>
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
</div>
`;
